src/script/backport-create-issue

```diff
@@ -78,7 +82,7 @@ def parse_arguments():
                         action="store_true")
     parser.add_argument("--dry-run", help="Do not write anything to Redmine",
                         action="store_true")
-    parser.add_argument("--force", help="Create backport issues even if status not Pending Backport",
+    parser.add_argument("--force", help="Create backport issues even if already processed or status not Pending Backport",
```

Comment:
Yeah, I didn't want to add another flag to this script and reuse the `--force` for different use cases, as you mentioned:
* When issue numbers provided, it ignores the Pending Backport status
* Otherwise it ignores the "backport_processed" tag.

What about something like:
```
 When issue numbers provided, process them even if not in 'Pending Backport' status. Otherwise,
 process them even if already processed (tag 'backport_processed' added)
```

Reply:
I suggested a slight amendment to the second sentence, otherwise looks good!

Reply:
Thanks! Already updated with you suggestion.